NEWS.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## [7.0.3] - 2016-05-25
```

Comment:
Don't put a new version number here, just the "Added"  section is sufficient.

This would be a minor version bump, anyway.